src/validators/rsavalidator.cpp

```diff
@@ -58,15 +58,15 @@ bool RSAValidator::Sign(const uint8_t *header, size_t num_header,
   size_t needed = 0;
   bool success = false;
 
-  EVP_MD_CTX evp_md_ctx;
-  EVP_MD_CTX_init(&evp_md_ctx);
-  EVP_DigestSignInit(&evp_md_ctx, NULL, md_, NULL, private_key_);
-  if (EVP_DigestSignUpdate(&evp_md_ctx, header, num_header) != 1) {
+  EVP_MD_CTX *evp_md_ctx = EVP_MD_CTX_new();
+  EVP_MD_CTX_init(evp_md_ctx);
```

Comment:
This isn't actually needed now because EVP_MD_CTX_new() already initializes everything the way it should be.